### ISSUE TITLE
Replace podUrl text input with a select input of user's contacts in "sharing" modals

### ISSUE THREAD

**Issue by xscottxbrownx** [codeforpdx/PASS#359]
### Describe the Current Behavior/Feature:
In both of the "sharing" modals in the `Profile` page, the user must type in the podUrl of another person they want to set the permissions for.

---
### Proposed Behavior/Feature:
Replace this with a dropdown select input of the user's contacts.

---
### Rationale:
Users will have a hard time remembering the podUrls of others since each server can have a different format, and given our clientele.

---
### Proposed Implementation (if applicable):
**1.** Create some sort of "contacts" list for each user
**2.** Populate the select menu options with said "contacts" list

---
### Additional context:
<img width="382" alt="Screenshot 2023-08-01 at 8 41 59 AM" src="https://github.com/codeforpdx/PASS/assets/71395931/82904af3-d85c-4651-9f53-bff7a54a5983">

**Comment by Raspber** [comment]
Going to give this a shot.

**Comment by Raspber** [comment]
working on this this week.

**Comment by xscottxbrownx** [comment]
@Raspber still interested in this?

**Comment by Raspber** [comment]
> @Raspber still interested in this?
> 

Yes, but if someone else would like to give this a shot, they can. I have not had a chance to begin this yet.
